### PR TITLE
[PLAY-5]  Add additional flex box property to  FlexItem kit

### PR DESCRIPTION
**Pull request by JelenaD1**
#### Screens

React_flex_item_props

Screenshot_11_17_21__9_56_AM

#### Breaking Changes
No. This change is adding a prop to FlexItem kit.

#### Runway Ticket URL
https://nitro.powerhrg.com/runway/backlog_items/PLAY-5

#### How to test this
Added tests for order prop in spec file. Navigate to kits/flex/rails and kits/flex/react to inspect the new prop example.

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

**Review by jasoncypret — APPROVED** [review]

**Review by jasoncypret — DISMISSED** [review]
I think the justified css is causing existing styling to break. We will need to investigate. CC @kre8sions

File: playbook/app/pb_kits/playbook/pb_flex/_flex_item.scss [hunk 1]
```diff
@@ -12,6 +12,37 @@
...
+  @for $i from 0 through 12 {
+    &[class*=order_#{$i}]{
+      order: $i;
+    }
+  }
+  &[class*=order_first]{
+    order: -1;
+  }
+
+  //Justify Items
```
Comment (kre8sions) [inline playbook/app/pb_kits/playbook/pb_flex/_flex_item.scss:25]: @JelenaD1 lets remove these justify Items styles

File: playbook/app/pb_kits/playbook/pb_flex/_flex.scss [hunk 2]
```diff
@@ -69,13 +69,15 @@ $gapTypes: (
     justify-content: center;
   }
 
+
   // Alignment: Align Items
-  &[class*=align_items_left],
-  &[class*=align_items_top],
-  &[class*=align_items_start] {
+  &[class*=justify_items_left],
```
Comment (kre8sions) [inline playbook/app/pb_kits/playbook/pb_flex/_flex.scss:74]: @JelenaD1 let's change this back to align_items

**Review by kre8sions — CHANGES_REQUESTED** [review]
We just need to fix the areas where the align-items is overwritten and delete the justify-items scss

File: playbook/app/pb_kits/playbook/pb_flex/_flex.scss (hunk 2 above)
Comment (JelenaD1) [reply playbook/app/pb_kits/playbook/pb_flex/_flex.scss:74]: I've made the changes but existing styling is still breaking on the page above. Any idea what to inspect next?

**Comment by jasoncypret** [comment]
LGTM I'll let @kre8sions add the final review

**Review by kre8sions — APPROVED** [review]

**Pull request merged**